dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A library that implements a layer on top of PKCS11 API"
+DESCRIPTION="The official header files for PKCS11 Standard were released \
+under a non-free license. libp11 does not include these header files, but \
+instead uses a rewritten pkcs11.h header file from the Scute project."
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="1991, 1999 Free Software Foundation"
```

Comment:
Should be
```
COPYRIGHT="2016 Michał Trojnara
	2015-2016 Nikos Mavrogiannopoulos
	2007-2009 Andreas Jellinghaus
	2011 Martin Paljak"
```
Most active authors appear first.
Source: [NEWS](https://github.com/OpenSC/libp11/blob/master/NEWS) file from the latest archive.